### DISCUSSION TITLE
Can the milvus master(version 2.0.0) branch develop/build/run on MacOS?

### DISCUSSION THREAD

**Discussion by dengc367** [milvus-io/milvus#5860] [Q&A and General discussion] [answered]
Hi Milvus Team, I am wonder that if the Milvus master branch can build/run on the MacOS without docker? If it can, please give me a doc link. Thanks. Because my laptop is MacOS.

**Comment by yhmo** [comment]
There are several components written by C++, including the index engine, so I think it cannot successfully build under MacOS.

**Reply by dengc367** [reply]
So I am curious about how to debug/develop Milvus with docker using Visual Code IDE in MacOS? becuase I believe  that there are many software developers using the MacOS in your Zilliz Company. And do you have such like a develop manual with vscode in public?

**Comment by yhmo** [comment, marked as answer]
Some developers compile milvus on MacOS by this way:
[compile_milvus2_macos.pdf](https://github.com/milvus-io/milvus/files/6690671/compile_milvus2_macos.pdf)

**Reply by dengc367** [reply]
Great!!!
